packages/gatsby/src/services/create-pages.ts

```diff
@@ -0,0 +1,70 @@
+import { IBuildContext } from "../state-machines/develop"
+
+import reporter from "gatsby-cli/lib/reporter"
+import apiRunnerNode from "../utils/api-runner-node"
+
+import {
+  deleteUntouchedPages,
+  findChangedPages,
+} from "../utils/check-for-changed-pages"
```

Comment:
This utility seems to be missing in branch

Reply:
Added